Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -1553,17 +1554,32 @@ def ip_command(client: Client, score_calculator: ScoreCalculator, args: dict, re
     """
     ips = argToList(args['ip'])
     results: List[CommandResults] = list()
+    execution_metrics = ExecutionMetrics()
     for ip in ips:
         raise_if_ip_not_valid(ip)
         try:
             raw_response = client.ip(ip, relationships)
+            if raw_response.get('error', {}).get('code') == "QuotaExceededError":
+                execution_metrics.quota_error += 1
+                result = CommandResults(readable_output=f'Quota exceeded for IP: {ip}').to_context()
+                results.append(result)
+                continue
         except Exception as exception:
             # If anything happens, just keep going
             demisto.debug(f'Could not process IP: "{ip}"\n {str(exception)}')
+            execution_metrics.general_error += 1
             continue
+        execution_metrics.success += 1
         results.append(
-            build_ip_output(client, score_calculator, ip, raw_response, argToBoolean(args.get('extended_data')))
+            build_ip_output(client, score_calculator, ip, raw_response, argToBoolean(args.get('extended_data'))).to_context()
```

Comment:
why did you add call to_context() here?

Reply:
There's a design issue with `return_results` where if you return an array of items (`CommandResults` or any of the new formats) It will return the object instead of the formatted version for the server.

Reply:
I can add a fix for it in this PR or open another one specifically for the CSP issue.